.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+const base = require('@open-web3/dev-config/config/eslint.cjs');
+
+module.exports = {
+  ...base,
+  ignorePatterns: [
+    '.eslintrc.cjs',
+    '.eslintrc.js',
+    '.github/**',
+    '.vscode/**',
+    '.yarn/**',
+    '**/build/*',
+    '**/coverage/*',
+    '**/node_modules/*'
+  ],
+  parserOptions: {
+    ...base.parserOptions,
+    project: ['./tsconfig.json']
+  },
+  rules: {
+    ...base.rules,
+    '@typescript-eslint/no-explicit-any': 'off'
+  }
+};
```

Comment:
This also seems like a copy-paste from somewhere, can't we just use recommended rules like here?: https://github.com/subspace/subspace/pull/16/files#diff-a154274cf11df18108900b95c7d92353cfacc900e604dbad8f8c03672b7cfa54

Reply:
It is, i was trying to get an idea on how to do this repository in a similar way to other projects and polkadot/api, i have tested this lint rules and actually  most of the are not useful, replaced with a basic recommended rules now. 

Also iam working in a branch much cleaner, updated and refactored for v14 metadata changes for subspace version, and v13 current spartan version. this branch remove lot of this initial cloned files and configurations, but will be sent in a separate PR after closing this.